### ISSUE TITLE
waterRAT not opening in Chrome

### ISSUE THREAD

**Issue by jlwilson-usgs** [bhuffman-usgs/waterrat#2]
I pulled some of the example data from the [Savannah DR](https://www.sciencebase.gov/catalog/item/5c8c21cfe4b0938824529ffd) and tried to get waterRAT up and running. 

Everything seems to be running correctly (my laptop fan kicks in which makes me think it is running) but when I try to open "localhost:8050" or "127.0.0.1:8050" in google chrome, I get: ERR_CONNECTION_REFUSED and ERR_CONNECTION_TIMED_OUT, respectively.

**Comment by jlwilson-usgs** [comment]
Per @bhuffman-usgs 's suggestion, I changed dz in the conf.ini file to 0.5.

My CPU did not get caught in an endless cycle and fan did not kick on. I also got additional output letting me know that the interface was running on http://127.0.0.1:8050/ but I am getting the same errors (RR_CONNECTION_REFUSED and ERR_CONNECTION_TIMED_OUT). 

![image](https://user-images.githubusercontent.com/8591248/64281229-4d9fe500-cf18-11e9-9f4e-1674b9e5c54b.png)
